### ISSUE TITLE
Use of SpecialPageFactory::getLocalNameFor was deprecated in MediaWiki 1.32. [Called from SMW\DataValues\TypesValue::makeSpecialPageTitleText ...]

### ISSUE THREAD

**Issue by mwjames** [SemanticMediaWiki/SemanticMediaWiki#4603]
### Setup and configuration

```
Using PHP 7.4.0
Semantic MediaWiki: 3.2.0-alpha, git: ad102f9
                    SMWSQLStore (mysql:5.6.33-0ubuntu0.14.04.1)
MediaWiki:          1.35.0-alpha, git: 8ea8171 (refs/heads/master)
                    Extension vendor autoloader
Site language:      en
Execution time:     2020-02-29 01:58
Debug logs:         Disabled
Xdebug:             Disabled (or not installed)
Intl/ICU:           7.4.0 / 52.1
PCRE:               10.33 2019-04-16
```

### Stack trace

```
1) SMW\Test\PropertiesQueryPageTest::testFormatPropertyItemOnUserDefinedProperty
Use of SpecialPageFactory::getLocalNameFor was deprecated in MediaWiki 1.32. [Called from SMW\DataValues\TypesValue::makeSpecialPageTitleText in /home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/src/DataValues/TypesValue.php at line 246]
/home/travis/build/SemanticMediaWiki/mw/includes/debug/MWDebug.php:333
/home/travis/build/SemanticMediaWiki/mw/includes/debug/MWDebug.php:265
/home/travis/build/SemanticMediaWiki/mw/includes/GlobalFunctions.php:1035
/home/travis/build/SemanticMediaWiki/mw/includes/specialpage/SpecialPageFactory_deprecated.php:91
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/src/DataValues/TypesValue.php:246
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/src/DataValues/TypesValue.php:170
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/includes/querypages/PropertiesQueryPage.php:202
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/includes/querypages/PropertiesQueryPage.php:146
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/includes/querypages/PropertiesQueryPage.php:103
/home/travis/build/SemanticMediaWiki/mw/extensions/SemanticMediaWiki/tests/phpunit/includes/querypages/PropertiesQueryPageTest.php:104
/home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php:99
```

https://travis-ci.org/SemanticMediaWiki/SemanticMediaWiki/jobs/656525350

**Comment by kghbln** [comment]
Direct cause of what was done as noted with #4623. Since SMW 3.2.0 is not planned to support MW 1.36 I added this to the SMW 3.3.0 milestone. Since the latter drops support for MW 1.31 not extra shim appears to be necessary to address this issue.

**Comment by Seb35** [comment]
This was probably solved together with #4602 since the line specified in the stacktrace `src/DataValues/TypesValue.php at line 246` was solved in #4782 – and the last occurence of `SpecialPageFactory::` is being fixed in #4943.
